static/assets/views/entities.html

```diff
@@ -0,0 +1,43 @@
+
+<h1>{{ctrl.title}} Entity Reference</h1>
+
+<p>
+    This topic describes the entity and type representations for common items that the {{ctrl.title}} API uses. To learn
+    about how to set up your developer account and work with the SKY API, see the
+    <a ng-href="{{ ctrl.getStartedUrl }}">Getting Started</a> guide.
+</p>
+
+<p ng-show="ctrl.showErrorMessage" class="alert alert-danger">
```

Comment:
Nice! Should we include the response status code or message? (maybe within an expander or something?)